### PR TITLE
Modify Canister Gas verb

### PR DESCRIPTION
**Pull request by ike709**
Adds a VV dropdown verb to canisters (requires DEBUG) to modify gasses in it. 

The way that gas contents work is that you don't actually *add* or *remove* a gas. You directly set the amount of a gas type in the mixture. In layman's terms this effectively means that everything has 0 moles of every gas unless set to a higher amount. So we set the moles of a chosen gas, then set the temperature for the mixture.

Demo: https://streamable.com/9m4ljh (Verb has since been moved from right click to VV dropdown)

## Changelog
:cl:
add: Admins can now easily modify a gas in a canister using VV dropdown.
/:cl:

### CODE REVIEW

**Comment by ike709** [comment]
bloons paid me $10 for this

**Comment by ike709** [comment]
DNM for a moment, the verb is appearing on objects it shouldn't.

**Comment by ike709** [comment]
Moved it from right click to VV dropdown.

**Review by PowerfulBacon — APPROVED** [review]
